lib/ddtrace/contrib/elasticsearch/core.rb

```diff
@@ -0,0 +1,61 @@
+require 'uri'
+require 'ddtrace/pin'
+require 'ddtrace/ext/app_types'
+
+URL = 'elasticsearch.url'.freeze
+METHOD = 'elasticsearch.method'.freeze
+TOOK = 'elasticsearch.took'.freeze
+PARAMS = 'elasticsearch.params'.freeze
+BODY = 'elasticsearch.body'.freeze
+
+DEFAULTSERVICE = 'elasticsearch'.freeze
+
+module Datadog
+  module Contrib
+    module Elasticsearch
+      # Elastic Search integration.
+      module TracedClient
+        def initialize(*args)
```

Comment:
On further reading and thought, I'm not totally sure what the pin onto and get_from fanciness add. Eg without:

```
def initialize(*args)
  self.datadog_trace_pin = Datadog::Pin.new(DEFAULTSERVICE, app: 'elasticsearch')
  super(*args)
end
```